app/assets/javascripts/compute_resources/kubevirt/kubevirt.js

```diff
@@ -0,0 +1,37 @@
+bootableRadio = function (item) {
+  const disabled = $('[id$=_bootable_true]:disabled:checked:visible');
+
+  $('[id$=_bootable_true]').prop('checked', false);
+  if (disabled.length > 0) {
+    disabled.prop('checked', true);
+  } else {
+    $(item).prop('checked', true);
+  }
+}
+
+cniProviderSelected = function (item) {
+  const selected = $(item).val();
+  if (selected === "pod") {
```

Comment:
can we simplify this by just doing:  toggleNetworkElement(selected != "pod") or something like that?